app/controllers/agentAppointment/AppointAgentController.scala

```diff
@@ -149,21 +147,25 @@ class AppointAgentController @Inject()(
       },
       success = (action: AgentAppointBulkAction) => {
         accounts.withAgentCode(action.agentCode.toString) flatMap {
-          case Some(group) => {
-            for {
-              _ <- Future.traverse(action.propertyLinkIds)(pLink =>
-                createAndSubmitAgentRepRequest(
-                  pLink,
-                  group.id,
-                  request.organisationId,
-                  request.individualAccount.individualId,
-                  action.checkPermission,
-                  action.challengePermission,
-                  request.organisationAccount.isAgent)).recover {
-                case e => Logger.warn(s"Failed to get a property link during multiple property agent appointment: ${e.getMessage}", e)
-              }
-            } yield
-              Ok(views.html.propertyrepresentation.appoint.appointAgentSummary(action, group.companyName))
+          case Some(group) => appointRevokeService.createAndSubmitAgentRepRequest(action.propertyLinkIds,
+            group.id,
```

Comment:
can we name the parameters here? from github unclear what is what.

Reply:
done